### PR TITLE
.NET 8 (Preview) support in Core Tools. 

### PR DESCRIPTION
**Pull request by khkh-ms**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: src/Azure.Functions.Cli/Actions/ContainerService/AzureContainerAppsDeployAction.cs [hunk 1]
```diff
@@ -108,7 +108,7 @@ public override async Task RunAsync()
             {
                 Location = environmentLocation;
             }
-            else if (environmentLocation != "eastasia" || !Location.Contains("eastasia"))
+            else if ((environmentLocation != "eastasia" && environmentLocation != "East Asia") || !Location.Contains("eastasia"))
```
[the remark this answers is not in the document]
Comment (kamperiadis) [reply src/Azure.Functions.Cli/Actions/ContainerService/AzureContainerAppsDeployAction.cs:111]: Can we have this change in a separate PR if it's not related to .NET 8?

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 2]
```diff
@@ -33,9 +33,6 @@
     <PublishReadyToRun>false</PublishReadyToRun>
     <PublishReadyToRunShowWarnings>false</PublishReadyToRunShowWarnings>
   </PropertyGroup>
-  <ItemGroup>
-    <None Remove="StaticResources\templatesv2.json" />
-  </ItemGroup>
```
Comment (kamperiadis) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:38]: Just for my own learning, why do we need to remove this?

Comment (khkh-ms) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:38]: It was already added in the next item group (just below this code) with correct details therefore this config is not being used.

**Review by kamperiadis — APPROVED** [review]
LGTM!

**Pull request merged**